src/api/environment.cc

```diff
@@ -665,9 +674,6 @@ Maybe<bool> InitializeContextForSnapshot(Local<Context> context) {
   Isolate* isolate = context->GetIsolate();
   HandleScope handle_scope(isolate);
 
-  context->AllowCodeGenerationFromStrings(false);
-  context->SetEmbedderData(
```

Comment:
We should initialize this to undefined at least, it's not quite likely that any code would be able to access it in a context that never runs through `InitializeContextRuntime()`, still it is possible if someone changes the bootstrap code.

Reply:
Thanks for the suggestion! Updated.